There is a bug in Tk versions 3.3-3.6 that causes a core dump
at line 1467 of file tkTextDisp.c under some esoteric conditions
where a text widget gets redisplayed when it has a -yscrollcommand
but hasn't been mapped onto the screen.  The "exmh" program seems
to be particularly adept at causing the problem, and many of you
have reported this problem, either on the newsgroup or directly
to me.  Here is a patch to tkTextDisp.c that should fix the
problem:

*** /tmp/,RCSt1346918	Mon Dec 13 17:00:28 1993
--- tkTextDisp.c	Mon Dec 13 17:00:15 1993
***************
*** 1217,1222 ****
--- 1217,1223 ----
      if ((textPtr->tkwin == NULL) || !Tk_IsMapped(textPtr->tkwin)
  	    || (dInfoPtr->maxX <= dInfoPtr->x)
  	    || (dInfoPtr->maxY <= dInfoPtr->y)) {
+ 	UpdateDisplayInfo(textPtr);
  	goto doScrollbars;
      }
      numRedisplays++;

